CHANGES.md

```diff
@@ -1,5 +1,31 @@
 # RxJava Releases #
 
+### Version 1.0.0 - August 5th 2015 ####
+
+Initial stable release!
+
+In order to provide a library that no project using RxJava would hesitate to depend on, the decision
+was made to remove the APIs which were not absolutely fundamental to all apps. That is what's
+contained in this release.
+
+Functionality which was previously part of this library is being explored in separate, modular
```

Comment:
Perhaps we should link RxBinding, RxLifecycle, and any other libs we know have directly taken on the roles that RxAndroid used to play? I know if I was reading this for the first time I'd be wondering where all my APIs went.